serializer-deserializer/pom.xml

```diff
@@ -106,6 +106,25 @@
             <artifactId>apicurio-registry-protobuf-schema-utilities</artifactId>
```

Comment:
Can we remove this since we will use the patching library?

Reply:
Is this addressed?

Reply:
I will address the following comments for Apicurio in the next PR.